src/ddmd/blockexit.d

```diff
@@ -36,34 +36,38 @@ import ddmd.visitor;
  */
 enum BE : int
 {
-    BEnone = 0,
-    BEfallthru = 1,
-    BEthrow    = 2,
-    BEreturn   = 4,
-    BEgoto     = 8,
-    BEhalt     = 0x10,
-    BEbreak    = 0x20,
-    BEcontinue = 0x40,
-    BEerrthrow = 0x80,
-    BEany      = (BEfallthru | BEthrow | BEreturn | BEgoto | BEhalt),
+    none      = 0,
+    fallthru  = 1,
+    _throw    = 2,
+    _return   = 4,
+    _goto     = 8,
+    halt      = 0x10,
+    _break    = 0x20,
+    _continue = 0x40,
+    errthrow  = 0x80,
+    any       = (fallthru | _throw | _return | _goto | halt),
 }
 
-alias BEnone = BE.BEnone;
-alias BEfallthru = BE.BEfallthru;
-alias BEthrow = BE.BEthrow;
-alias BEreturn = BE.BEreturn;
-alias BEgoto = BE.BEgoto;
-alias BEhalt = BE.BEhalt;
-alias BEbreak = BE.BEbreak;
-alias BEcontinue = BE.BEcontinue;
-alias BEerrthrow = BE.BEerrthrow;
-alias BEany = BE.BEany;
+alias BEnone     = BE.none;
+alias BEfallthru = BE.fallthru;
+alias BEthrow    = BE._throw;
+alias BEreturn   = BE._return;
+alias BEgoto     = BE._goto;
+alias BEhalt     = BE.halt;
+alias BEbreak    = BE._break;
+alias BEcontinue = BE._continue;
+alias BEerrthrow = BE.errthrow;
+alias BEany      = BE.any;
 
 
 
 /*********************************************
- * Only valid after semantic analysis
+ * Determine mask of ways that a statement can exit.
+ *
+ * Only valid after semantic analysis.
  * Params:
+ *   s = statement to check for block exit status
+ *   func = function that statement s is in
  *   mustNotThrow = generate an error if it throws
  * Returns:
  *   BExxxx
```

Comment:
Maybe that part of the doc can be updated as well since we have a proper enum now